charts/unifi/templates/deployment.yaml

```
@@ -223,6 +223,9 @@ spec:
         {{- with .Values.extraVolumes  }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+      {{- with .Values.priorityClassName }}
+      priorityClassName: {{ toYaml . | quote | nindent 8 }}
```

```suggestion
      priorityClassName: {{ . }}
```

We don't need toYaml or quote here since it is a field with a basic string type.